src/components/Home/Home.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import './Home.scss'
+
+const Home = () => {
+    return (
+      <section className="homepage">
+        <h1 className="homepageTitle">Welcome to PenPost</h1>
+        <h2>About Us</h2>
+        <p>How it works</p>
+        <p>
```

Comment:
I think you brought up a great point about it not being vey exciting. I'm sure there's some simple things that we can do to ensure that it's livened up a little. Maybe a postage stamp with PP inside of it somewhere?